mycore-oai/src/main/java/org/mycore/oai/set/MCROAISolrSetConfiguration.java

```diff
@@ -44,8 +44,7 @@ public MCROAISolrSetConfiguration(String configPrefix, String setId) {
         String setConfigPrefix = configPrefix + SETS_PREFIX + setId;
         MCROAISolrSetHandler handler = MCRConfiguration2.getInstanceOf(
                 MCROAISolrSetHandler.class, setConfigPrefix + ".Handler")
-            .orElseGet(() -> MCRConfiguration2.getInstanceOfOrThrow(
-                MCROAISolrSetHandler.class, getFallbackHandler(configPrefix, setId)));
+            .orElseGet(() -> MCRConfiguration2.instantiateClass(getFallbackHandler(configPrefix, setId)));
```

Comment:
The fallback handler might be be `null`, support for which was unintentionally dropped in MCR-3053.

Maybe:

```
        MCROAISolrSetHandler handler = MCRConfiguration2
            .getInstanceOf(MCROAISolrSetHandler.class, setConfigPrefix + ".Handler")
            .orElseGet(() -> {
                String fallbackHandler = getFallbackHandler(configPrefix, setId);
                if (fallbackHandler == null) {
                    throw new MCRConfigurationException("Configuration property " +  setConfigPrefix 
                        + ".Handler is not set and no fallback handler could be determined");
                }
                return MCRConfiguration2.instantiateClass(fallbackHandler);
            });
```